invokeai/app/invocations/image_to_latents.py

```diff
@@ -130,6 +143,11 @@ def invoke(self, context: InvocationContext) -> LatentsOutput:
         assert isinstance(vae_info.model, (AutoencoderKL, AutoencoderTiny))
```

Comment:
Improve error reporting when an incompatible VAE is input. Currently reports "Server Error" with no details. If accepted, I suggest making similar change at line 78.
```suggestion
        assert isinstance(vae_info.model, (AutoencoderKL, AutoencoderTiny)), "VAE must be of type SD-1.5 or SDXL"
```

Reply:
Yeah, I think those original asserts are from back before there were any other VAE types. Added the warning text to all instances.